.github/workflows/pr-checks.yml

```
@@ -12,7 +12,8 @@ on:
     types:
       - milestoned
       - demilestoned
-
+concurrency:
+  group: pr-checks
```

If I understand this correctly it will only allow one pr check to run at a time. Can we make it so that one concurrent pr check per issue/pr to run at a time? Something like below based on https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency and https://docs.github.com/en/actions/learn-github-actions/contexts#github-context is maybe possible?

```suggestion
  group: pr-checks-${{ github.event.pull_request.number || github.event.issues.issue.number }}
```